commitlint.config.js

```diff
@@ -0,0 +1,41 @@
+// Configuration is extended from @commitlint/config-conventional
+// @see https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional
+module.exports = {
+	rules: {
```

Comment:
This is probably a stretch ask. Is there a way to enforce an issue number to be associated with the commit message? like

#123 - fix(spec): ambiguity around update operation terminology

Reply:
There are integrations like this but I would have to investigate a little more

Reply:
code reviews can address this. just reject the PR if it doesn't reference issues.